mint-auditor/src/db/mod.rs

```diff
@@ -354,15 +371,15 @@ mod tests {
         for block_index in 0..initial_num_blocks {
             let block_data = ledger_db.get_block_data(block_index).unwrap();
 
-            let (mint_audit_data, balance_map) = mint_audit_db
+            let sync_block_data = mint_audit_db
                 .sync_block(block_data.block(), block_data.contents())
                 .unwrap();
 
             assert_eq!(
-                mint_audit_data,
+                sync_block_data.block_audit,
                 BlockAuditData::new(block_data.block().index),
             );
-            assert_eq!(balance_map, Default::default());
+            assert_eq!(sync_block_data.balance_map, Default::default());
```

Comment:
Good idea, I'll give it a shot

Reply:
So one note I have on this after making the change, is that having the assert_eq! look at the mint_txs is not really doable in a meaningful way, since I need to query the database to get them. I know what the expected mint_txs sans their auto-generated id, so if I wanted to compare them with what I expect to be there I will need to ignore the id field (or make hardcoded assumptions on what ids they end up with, which feels janky)

Reply:
I am going to leave this as-is for now, this can be improved in the future if we decide to invest more time in it.